### ISSUE TITLE
Quick question...

### ISSUE THREAD

**Issue by mbrucedogs** [spektor56/KaraokePlayer#1]
I am a developer and was looking to build my own karaoke solution to play local files CDG and MP4 using VLC.  I was looking on GITHub to see if someone had done any of this work yet which is when I ran across this project last night. 

I wasn't sure on how far along this project was since for some reason I can't seem to get it to build.  If you have a chance reply to this and let me know if this project is something I should even try to work with or Fork off of?

Thanks.

**Comment by spektor56** [comment]
I just started this project around 3 or so days ago.  I have the CDG file rendering working with a synced VLC player with visualizations as the underlay.  I'm in the process of rewriting the CDG code, check back in a week or so and it should be in a buildable state. 

I took the original CDG code from: http://www.codeproject.com/Articles/89845/MP-CDG-Karaoke-Player

I converted it from VB.Net to C# and i'm in the process of rewriting it using the CDG documentation at: http://jbum.com/cdg_revealed.html

If you want to wait when this project is done you will be able to play any video file or use any mp3 visualization preset you want behind the CDG lyrics. (This already works but i'm just refactoring the code)
